homeassistant/components/conversation/__init__.py

```diff
@@ -115,9 +115,10 @@
             {
                 vol.Optional("intents"): vol.Schema(
                     {cv.string: vol.All(cv.ensure_list, [cv.string])}
-                )
+                ),
+                vol.Optional("fuzzy_matching"): bool,
```

Comment:
Why do we allow disabling it ?

Reply:
That part was left over from my original implementation. I'm change the agent's property directly in the actual test fixtures now.